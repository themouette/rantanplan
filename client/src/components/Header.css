.App-Header {
  background-color: #222;
  color: white;
  text-align: center;
}

.App-Title {
  text-align: start;
  display: inline-block;
  padding: 20px;
}
.App-Logo {
  display: inline-block;
  height: 60px;
}

.App-Title-Content {
  font-size: 1.5em;
}
.App-Title-Subtitle {
  font-size: 1.2em;
  display: none;
}

@media screen and (min-width: 640px) {
  .App-Logo {
    height: 80px;
  }

  .App-Title-Content {
    font-size: 1.5em;
  }

  .App-Title-Subtitle {
    display: block;
  }
}
