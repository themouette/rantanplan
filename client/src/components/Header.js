/* @flow */
import React, { Component } from 'react';

import './Header.css';
import logo from '../assets/rantanplan-250.png';


type Props = {};

/**
 * Header component
 */
class Header extends Component<Props> {
  static defaultProps = {};

  render() {
    return (
      <header className="App-Header">
        <img src={logo} className="App-Logo" alt="Rantanplan" />
        <div className="App-Title">
          <h1 className="App-Title-Content">
            Rantanplan
          </h1>
          <h2 className="App-Title-Subtitle">
            Modern Monitoring & Analytics (For Localhost)
          </h2>
        </div>
      </header>
    );
  }
}

export default Header;
