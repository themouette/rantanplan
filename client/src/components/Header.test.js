import React from 'react';
import Header from './Header';
import { shallow } from 'enzyme';

describe('<Header />', () => {
  it('should render', () => {
    const wrapper = shallow(<Header />);

    expect(wrapper).toMatchSnapshot();
  });
});
