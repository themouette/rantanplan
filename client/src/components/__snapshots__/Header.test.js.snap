// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> should render 1`] = `
<header
  className="App-Header"
>
  <img
    alt="Rantanplan"
    className="App-Logo"
    src="rantanplan-250.png"
  />
  <div
    className="App-Title"
  >
    <h1
      className="App-Title-Content"
    >
      Rantanplan
    </h1>
    <h2
      className="App-Title-Subtitle"
    >
      Modern Monitoring & Analytics (For Localhost)
    </h2>
  </div>
</header>
`;
